dojo/finding/views.py

```diff
@@ -1542,143 +2024,174 @@ def apply_cwe_mitigation(apply_to_findings, template, update=True):
                 apply_message = ""
 
             messages.add_message(
                 request,
                 messages.SUCCESS,
-                'Template ' + apply_message + 'updated successfully.',
-                extra_tags='alert-success')
-            return HttpResponseRedirect(reverse('templates'))
+                "Template " + apply_message + "updated successfully.",
+                extra_tags="alert-success",
+            )
+            return HttpResponseRedirect(reverse("templates"))
         else:
             messages.add_message(
                 request,
                 messages.ERROR,
-                'Template form has error, please revise and try again.',
-                extra_tags='alert-danger')
+                "Template form has error, please revise and try again.",
+                extra_tags="alert-danger",
+            )
 
     count = apply_cwe_mitigation(True, template, False)
     add_breadcrumb(title="Edit Template", top_level=False, request=request)
-    return render(request, 'dojo/add_template.html', {
-        'form': form,
-        'count': count,
-        'name': 'Edit Template',
-        'template': template,
-    })
+    return render(
+        request,
+        "dojo/add_template.html",
+        {
+            "form": form,
+            "count": count,
+            "name": "Edit Template",
+            "template": template,
+        },
+    )
 
 
 @user_has_global_permission(Permissions.Finding_Delete)
 def delete_template(request, tid):
     template = get_object_or_404(Finding_Template, id=tid)
+    # Remove this assignment to local variable 'form'; the value is never used
+    # form = DeleteFindingTemplateForm(instance=template)
 
-    form = DeleteFindingTemplateForm(instance=template)
-
-    if request.method == 'POST':
+    if request.method == "POST":
         form = DeleteFindingTemplateForm(request.POST, instance=template)
         if form.is_valid():
             template.delete()
             messages.add_message(
                 request,
                 messages.SUCCESS,
-                'Finding Template deleted successfully.',
-                extra_tags='alert-success')
-            return HttpResponseRedirect(reverse('templates'))
+                "Finding Template deleted successfully.",
+                extra_tags="alert-success",
+            )
+            return HttpResponseRedirect(reverse("templates"))
         else:
             messages.add_message(
                 request,
                 messages.ERROR,
-                'Unable to delete Template, please revise and try again.',
-                extra_tags='alert-danger')
+                "Unable to delete Template, please revise and try again.",
+                extra_tags="alert-danger",
+            )
     else:
         return HttpResponseForbidden()
 
 
-def download_finding_pic(request, token):
+# Remove the unused function parameter "request"
+# TODO : REVISAR
+def download_finding_pic(token):
```

Comment:
I believe the request object needs to be there whether it is used or not. Probably a Django requirement. If Pep8/Black is complaining about it not being used, the we could add some check to make sure the path is what is expected and add a comment explaining the reason for the useless check

Reply:
ok